rust-symcrypt/BUILD.md

```diff
@@ -69,5 +71,5 @@ After building SymCrypt on linux, the required `libsymcrypt.so*` files can be fo
 
 The symcrypt crate needs to be able to link with these libs during build/run time. In order to mimic the installation path for other libraries, you must place the `libsymcrypt.so*` files into the `/usr/lib/x86_64-linux-gnu/` folder. This is where the linker will look for the `libsymcrypt.so*` files. 
 
-**Note:** This process has been streamlined for `Mariner`, and there is no garuntee for combatibility on other Linux distros at the moment. 
+**Note:** This process has been streamlined for `Mariner`, and there is no guarantee for compatibility on other Linux distros at the moment. 

```

Comment:
For Azure Linux, libsymcrypt.so should be installed by default. At least, that's the case for Azure Linux 3. In 2 you might need to install it via package manager? @samuel-lee-msft might know.

More general guidance is that libsymcrypt.so needs to be placed in the linker load path. Unfortunately the way this is set varies between distros. On most distros it's set in the environment variable `$LD_LIBRARY_PATH`. On Ubuntu, it's set in this file: `/etc/ld.so.conf.d/x86_64-linux-gnu.conf`

It's still true that we don't officially support distributions other than Azure Linux and Ubuntu.

Reply:
> For Azure Linux, libsymcrypt.so should be installed by default. At least, that's the case for Azure Linux 3. In 2 you might need to install it via package manager? @samuel-lee-msft might know.

I spoke with @mamckee  yesterday and from our conversation it seems that Azure Linux 3 comes out the box, and for Azure Linux 2 there is a command similar to `sudo apt get` to get the `symcrypt` package 

With @mamckee's work on the Debian project do you think it might be simpler to just remove this Linux section and just say that a Debian package is coming soon?

Reply:
Current wording looks good to me, but maybe we can add instructions for Azure Linux?

For Azure Linux 2.0 it should just be `dnf install symcrypt` (note: I haven't tested this myself)

In 3.0 it will be included in-box, but 3.0 hasn't shipped yet

Reply:
I've updated the `BUILD.MD` to have an install section for Azure Linux, and I've tried the `tdnf install symcrypt` before on my machine and it works. I've also included a note about Azure Linux 3.0